dev/NavigationView/NavigationView.cpp

```diff
@@ -4290,14 +4523,6 @@ template<typename T> T NavigationView::GetContainerForData(const winrt::IInspect
         return nvi;
     }
 
-    if (auto settingsItem = m_settingsItem.get())
-    {
-        if (settingsItem == data || settingsItem.Content() == data)
```

Comment:
This seems to imply that there was a scenario where data would be the 'Settings' string and this function would return the settings item. To keep this scenario working, we probably shouldn't delete this.

Reply:
now settings is part of FooterRepeater, so it should be returned properly

Reply:
ok. Noticed that i didn't add any code for footer here.